Engine/source/T3D/player.cpp

```diff
@@ -4994,6 +4995,18 @@ void Player::_handleCollision( const Collision &collision )
       queueCollision( collision.object, mVelocity - collision.object->getVelocity() );
 }
 
+bool Player::_physicsEnabled()
```

Comment:
This looks like something very expensive to do considering that it'll be called multiple times per Player object instance per tick time.